data/human/weapons.txt

```diff
@@ -85,6 +85,28 @@ outfit "Quad Blaster Turret"
 		"hull damage" 6.6
 	description "A Quad Blaster Turret carries four guns on a single turret mount, doubling the firepower of an ordinary Blaster Turret for ships with enough space to install one and energy to drive it."
 
+outfit "Twin Blaster"
+	category "Guns"
+	cost 40000
+	thumbnail "outfit/twin blaster"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -10
+	"gun ports" -1
```

Comment:
```suggestion
	"mass" 13
	"outfit space" -13
	"weapon capacity" -13
	"gun ports" -1
```

This put its DPS/mass at 13.23, which is between the Proton Gun at 14.86 and the Particle Cannon at 12.7, which seems about right. 
Currently, its DPS/mass is above the heavy laser and just below the beam laser, which doesn't seem appropriate.

This also puts it directly between the Beam Laser at 8t and the Heavy Laser at 15t